ctapipe/image/cleaning.py

```diff
@@ -2,15 +2,20 @@
 Image Cleaning Algorithms (identification of noisy pixels)
 """
 
-__all__ = ['tailcuts_clean', 'dilate']
+__all__ = ["tailcuts_clean", "dilate"]
```

Comment:
just noticed that `__all__` doesn't include all the public functions in this module.  Please add `mars_cleaning_first_pass`, and while you are at it  `apply_time_delta_cleaning` and `fact_cleaning` to this list.

Reply:
OK!

Reply:
I think apply time delta cleaning was not supposed to be public, as it's only used by the FACT cleaning, right @LukasNickel ?

Reply:
I wish to be able to say that I didn't add it because I also thought of this, but honestly I forgot to add it. Good to know that it wasn't sure, then!

Reply:
Could you rename it with a leading underscore?

Reply:
Done!

Reply:
> I think apply time delta cleaning was not supposed to be public, as it's only used by the FACT cleaning, right @LukasNickel ?

Yeah, its just a helper function